### DISCUSSION TITLE
work with usb-drive

### DISCUSSION THREAD

**Discussion by maxkz** [mhammond/pywin32#2520] [General]
I want to manipulate with usb flash drive (get information about usb, enable, disable) using pywin32. But i can't find any wiki about this. Why so pure documentation? Why not have wiki about work with usb flash drive?

**Comment by saiyanx** [comment]
Doing something similar, found this on stackoverflow, hope its helpful

#https://stackoverflow.com/questions/8110310/simple-way-to-query-connected-usb-devices-info-in-python
import win32com.client

wmi = win32com.client.GetObject ("winmgmts:")
for usb in wmi.InstancesOf ("Win32_USBHub"):
    print(usb.DeviceID)

**Comment by Avasam** [comment]
https://github.com/mhammond/pywin32#support
> However, please **do not open github issues for general support requests**, or
for problems or questions using the modules in this package - they will be
closed. For such issues, please email the
[python-win32 mailing list](http://mail.python.org/mailman/listinfo/python-win32) -
note that you must be subscribed to the list before posting.
